modules/dns/README.md

```diff
@@ -0,0 +1,68 @@
+# CIS DNS Module
+
+This module is used to create and manage CIS DNS Records.
+
+**Note:**
+
+1. For SRV record, you need to provide record name repeatedly as shown in the below example.
+
+    ```
```

Comment:
Because this is no longer under a list item, you can remove the indentation on the example.

Reply:
done